AssertionError: Invalid DISPLAY :0 - cannot find X server with xdpyinfo

When I run ```python scripts/alfworld-play-thor```,  the code reports the following error, what should I do to solve the problem. 
Thanks!!!!!!

```python
Playing './json_2.1.1/train/pick_two_obj_and_place-Mug-None-Cabinet-318/trial_T20190910_055356_172564'.
No protocol specified
xdpyinfo:  unable to open display ":0".
Traceback (most recent call last):
  File "scripts/alfworld-play-thor", line 109, in <module>
    main(args)
  File "scripts/alfworld-play-thor", line 43, in main
    env = ThorEnv()
  File "/home/hongbo.wang/miniconda3/envs/alfworld/lib/python3.6/site-packages/alfworld/env/thor_env.py", line 39, in __init__
    player_screen_width=player_screen_width)
  File "/home/hongbo.wang/miniconda3/envs/alfworld/lib/python3.6/site-packages/ai2thor/controller.py", line 855, in start
    self.check_x_display(env['DISPLAY'])
  File "/home/hongbo.wang/miniconda3/envs/alfworld/lib/python3.6/site-packages/ai2thor/controller.py", line 715, in check_x_display
    ("Invalid DISPLAY %s - cannot find X server with xdpyinfo" % x_display)
AssertionError: Invalid DISPLAY :0 - cannot find X server with xdpyinfo
```

--- a/alfworld/gen/constants.py
+++ b/alfworld/gen/constants.py
@@ -1,4 +1,5 @@
 from collections import OrderedDict
+import os
 
 ########################################################################################################################
 # General Settings
@@ -85,7 +86,7 @@ data_dict = OrderedDict()  # dictionary for storing trajectory data to be dumped
 # Unity Hyperparameters
 
 BUILD_PATH = None
-X_DISPLAY = '0'
+X_DISPLAY = None
 
 AGENT_STEP_SIZE = 0.25
 AGENT_HORIZON_ADJ = 15

